src/config.h

```diff
@@ -52,11 +52,11 @@ struct x509_digest {
 #define COOKIE_SIZE	240
 
 struct vpn_config {
-	char 		gateway_host[FIELD_SIZE];
```

Comment:
BTW buf size of 64 is a bit small.
this common password would not work: {{=&$@#=&$@#=&$@#=&$@#!!}}
:)